index.ts

```diff
@@ -4,6 +4,8 @@ import * as Github from '@actions/github';
 
 const ghClient = process.env.GITHUB_TOKEN && new Github.GitHub(process.env.GITHUB_TOKEN);
 
+const { GITHUB_REPOSITORY = '' } = process.env;
```

Comment:
☝️ are we defaulting to an empty string because typescript assumes it's `string | undefined`?

Reply:
Yes, AFAIK GITHUB_REPOSITORY should always be defined, an alternative could be:

```
const { GITHUB_REPOSITORY } = process.env;

if(GITHUB_REPOSITORY === undefined) throw new Error("GITHUB_REPOSITORY environment variable is undefined.");
```

Reply:
Related docs:
https://docs.github.com/en/actions/reference/environment-variables

Reply:
yup that's what I assumed too, wanted to double check to make sure 😄 looks good!